tuf-spec.md

```diff
@@ -462,6 +462,19 @@ repo](https://github.com/theupdateframework/specification/issues).
    Delegated target roles are authorized by the keys listed in the directly
    delegating target role.
 
+* **3.2 Repository metadata creation**
+
+   Metadata SHOULD be generated in the following sequence, in order to ensure
+   that metadata are not referenced in the repository before they have been
+   created.  The below sequence assumes that all targets files referenced by
+   the metadata are available to the repository before the metadata is written.
+
+     * **3.2.1** delegated targets metadata (DELEGATED_ROLE.EXT)
+     * **3.2.2** root metadata (root.EXT)
+     * **3.2.3** top-level targets metadata (targets.EXT)
+     * **3.2.4** snapshot metadata (snapshot.EXT)
+     * **3.2.5** timestamp metadata (timestamp.EXT)
```

Comment:
Oh yes, I forgot about mirror. I think it _could_ be written at any point after the root metadata, because it's not actually referenced by or referencing any metadata explicitly. 

It may make sense to list it as 3.2.6, because then we can have some confidence that an implementation conforming to the spec won't write any mirrors metadata until all of the metadata it is mirroring has been written?

Reply:
Listing it as 3.2.6 makes sense to me. We can also mention it in the paragraph above to state that it can be written at any point.